### PR TITLE
22.2.22 project

### PR DESCRIPTION
**Pull request by rinky009**
@anita0911 check my code

### CODE REVIEW

File: 24.2.22 syncandasync/callback1.js [hunk 1]
```diff
@@ -0,0 +1,10 @@
+const funA=()=>{
+    setTimeout(function(){
+        console.log('welcome fun1');
```
Comment (anita0911) [inline 24.2.22 syncandasync/callback1.js:3]: put the brace properly....@rinky009

File: 24.2.22 syncandasync/callback3.js [hunk 2]
```diff
@@ -0,0 +1,10 @@
+const personOne=(friend,callfrnd)=>{
+    console.log(`I am busy right now.\nI am talking to ${friend}.\nI will call you back.`);
```
Comment (anita0911) [inline 24.2.22 syncandasync/callback3.js:2]: \nI am -define in a clear manner

File: 22.2.22object/spread1.js [hunk 3]
```diff
@@ -0,0 +1,5 @@
+//spread opreator
+const numbersOne=[1,2,3];
```
Comment (anita0911) [inline 22.2.22object/spread1.js:2]: good work....@rinky009

File: 22.2.22object/this.js [hunk 4]
```diff
@@ -0,0 +1 @@
+console.log(this);
```
Comment (anita0911) [inline 22.2.22object/this.js:1]: i am not getting what is this programm about@rinky009

File: 18-2-22 function/mapfilred.js [hunk 5]
```diff
@@ -0,0 +1,7 @@
+const arr=[1,2,3,4,5,6,7];
+const newArr=arr.filter(a=>a%2===0);
```
Comment (anita0911) [inline 18-2-22 function/mapfilred.js:2]: optimize your code....

File: 18-2-22 function/reduce2.js [hunk 6]
```diff
@@ -0,0 +1,3 @@
+const nums=[4,8,3,9,1]
+const max =nums.reduce((a,b) => b>a?b:a);
```
Comment (anita0911) [inline 18-2-22 function/reduce2.js:2]: good ....

**Review by anita0911 — APPROVED** [review]

File: 22.2.22object/spread1.js (hunk 3 above)
Comment (rinky009) [reply 22.2.22object/spread1.js:2]: @anita0911 practicing just

File: 24.2.22 syncandasync/callback3.js (hunk 2 above)
Comment (rinky009) [reply 24.2.22 syncandasync/callback3.js:2]: @anita0911 ok

File: 24.2.22 syncandasync/callback1.js (hunk 1 above)
Comment (rinky009) [reply 24.2.22 syncandasync/callback1.js:3]: @anita0911 i will take care

**Pull request merged**